civictechprojects/static/css/partials/_SplashScreen.scss

```diff
@@ -0,0 +1,44 @@
+.SplashScreen-root {
+  background-color: $color-text-grey;
+  color: $color-text-light;
+  width: 100%;
+  height: 80vh;
+}
+
+.SplashScreen-content {
+  position: relative;
+  text-align: center;
+  transform: translateY(-50%);
+  top: 50%;
+}
+
+.SplashScreen-section {
+  padding-bottom: 15px;
+}
+
+.SplashScreen-find-projects-btn {
```

Comment:
Add `margin-bottom: 10px;` here for half of the narrow-phone button stacking issue...